flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/flip1/partitionrelease/RegionPartitionReleaseStrategy.java

```diff
@@ -174,9 +174,9 @@ private boolean isRegionOfVertexFinished(final ExecutionVertexID executionVertex
 	public static class Factory implements PartitionReleaseStrategy.Factory {
 
 		@Override
-		public PartitionReleaseStrategy createInstance(final SchedulingTopology<?, ?> schedulingStrategy) {
+		public PartitionReleaseStrategy createInstance(final SchedulingTopology schedulingStrategy) {
 
-			final Set<? extends Set<? extends SchedulingExecutionVertex<?, ?>>> distinctRegions =
+			final Set<? extends Set<? extends SchedulingExecutionVertex>> distinctRegions =
```

Comment:
Now it can be simplified to `Set<Set<SchedulingExecutionVertex>>`.

Reply:
Ok, I found one more occurrence in `PipelinedRegionComputeUtil`